ons_ras_common/ons_rest_case.py

```diff
@@ -0,0 +1,244 @@
+##############################################################################
+#                                                                            #
+#   Generic Configuration tool for Micro-Service environment discovery       #
+#   License: MIT                                                             #
+#   Copyright (c) 2017 Crown Copyright (Office for National Statistics)      #
+#                                                                            #
+##############################################################################
+#
+#   ONSCase wraps routines used to access the case service
+#
+##############################################################################
+
+
+class ONSCase(object):
+    """
+    This class is designed to take all the work out of accessing the case service. Initially it
```

Comment:
What work is there in accessing the case service? Surely all we're doing is hitting an endpoint and getting a response? Is there a deficiency in the case service API that this code works around?